Example/MoyaComparison/MoyaComparison/Utils/WebServiceFactory.swift

```diff
@@ -0,0 +1,49 @@
+//
+// Copyright 2017 niji
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+import Foundation
+import NetworkStack
+import Moya
+
+struct WebServiceFactory {
+  static func clients() -> WebServiceClients {
+    let clients = WebServiceClients(
+      niji: NijiVideoWebServiceClient(services: services()),
+      moya: MoyaVideoWebServiceClient(services: services()),
```

Comment:
Is it intentional here that you *re*-create a `Services` instance every time? Instead of reusing the same for all WSClients? What about this instead, would it have any side-effect of sharing NetworkStacks between them? (I mean, there might be, like they'll then share accessTokens or stuff like that if they use the same `NetworkStack` instance, just wondering if it's problematic or not)

```swift
let services = services()
return WebServiceClients(
  niji: NijiVideoWebServiceClient(services: services),
  moya: NijiVideoWebServiceClient(services: services),
  …
)
```